shopify-client/src/components/Header.vue

```diff
@@ -35,9 +39,15 @@
 <script>
 export default {
   name: "Header",
+  methods:{
+    submitSearch: function(){
+      if(this.keyword){
+        window.location.href = '/app/shops/List?keyword=' + this.keyword;
```

Comment:
What was the decision in the end about changing this to `$router` instead of `window.location.href` for navigation?

Reply:
changing to $router
will push later